helm/vertical-pod-autoscaler-app/Chart.yaml

```diff
@@ -4,14 +4,19 @@ description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 icon: https://s.giantswarm.io/app-icons/k8s-initiator/1/dark.svg
 home: https://github.com/giantswarm/vertical-pod-autoscaler-app
 maintainers:
-- name: sudermanjr
+  - name: giantswarm/team-atlas
+    email: team-atlas@giantswarm.io
 name: vertical-pod-autoscaler-app
 sources:
-- https://github.com/kubernetes/autoscaler
+- https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler
```

Comment:
Can you add this link to the sources, 
`https://github.com/cowboysysop/charts/tree/master/charts/vertical-pod-autoscaler`
as well)

Reply:
Done